dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1062,6 +1063,29 @@ void TeachingTip::OnActionButtonClicked(const winrt::IInspectable&, const winrt:
 
 void TeachingTip::OnPopupOpened(const winrt::IInspectable&, const winrt::IInspectable&)
 {
+#ifdef USE_INSIDER_SDK
```

Comment:
>USE_INSIDER_SDK [](start = 7, length = 15)

Is USE_INSIDER_SDK really still needed for IUIElement10?  I thought that was in the public SDK by now?

Reply:
Public as of a few days ago. We haven't switched the project over to the "official" one yet. I just added a usage of uiElement10 in another PR and didn't add the conditional (I forgot). I would not bother adding it here either, but there's no particular harm.